### PR TITLE
add_divise

### PR DESCRIPTION
**Pull request by whiteWolfx99**
# Devise
### In this branch we have done the followings:

- add-devise
- added login and log out.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,10 +1,16 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (rloterh) [inline app/models/user.rb:5]: - [ ] Devise `:confirmable` option is not enabled. In this activity, we want to **ask for confirmation of email** from a user during a new registration process. Let's include this option by enabling devise `:confirmable` option.

File: app/views/devise/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
```
Comment (rloterh) [inline app/views/devise/sessions/new.html.erb:2]: - [ ] This activity requires the modification of the views in _devise_ for the registration and login to match the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly, like seen in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md):

**Your login view:**        |  **Wireframe login view:**
:-------------------------:|:-------------------------:
Home Page | Home Page

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Hevar, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🏇  

## Highlights 👍  
- Authentication setup using devise gem is coming up well 😉
- ReadMe looks great!

## Required Changes ♻️
_Kindly check the inline comments under the review._
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @whiteWolfx99  @dialite , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great devise setup ✔️ 
- No rubocop errors ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**